common/app/views/support/cleaner/VideoEmbedCleaner.scala

```diff
@@ -128,6 +132,11 @@ case class VideoEmbedCleaner(article: Article, maxEmbedHeight: Int = 812) extend
     document
   }
 
+  private def addVimeoDntFlag(iframe: Element) {
+    val src = iframe.attr("src")
+    iframe.attr("src", src ++ (if (src.contains("?")) "&" else "?") ++ "dnt=true")
```

Comment:
Is there a way to detect if consent has been given and change the value of that flag accordingly?